rust/feature-flags/src/api/errors.rs

```diff
@@ -53,6 +53,8 @@ pub enum FlagError {
     DatabaseError(sqlx::Error, Option<String>),
     #[error("Timed out while fetching data")]
     TimeoutError(Option<String>),
+    #[error("Query timeout: {0}")]
+    Timeout(&'static str),
```

Comment:
How does this `Timeout` differ from `TimeoutError`? Could they be combined or this one be renamed to be more specific about the type of timeout?